README.md

```diff
@@ -123,7 +126,6 @@ No deployment instructions yet.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228914508-7db96e74-6a10-4909-a0af-f6318f131823.png)

- [ ] Kindly put your name in place of Author1 in the Authors section because you are the author of this repo not Author1 😄